pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -85,6 +79,44 @@ class IvyFileWriter {
 
         def ivyText = writer.toString()
 
-        new File(destDir, "${name}-${version}.ivy").text = ivyText
+        if (packageType == SdistDownloader.SOURCE_DIST_PACKAGE_TYPE) {
+            new File(destDir, "${name}-${version}.ivy").text = ivyText
+        } else if (packageType == WheelsDownloader.BINARY_DIST_PACKAGE_TYPE) {
+            new File(destDir, "${name}-${version}-${classifier}.ivy").text = ivyText
+        }
+    }
+
+    private getPublicationsMap() {
+        def publicationMap = archives.collect { artifact ->
+            def ext = artifact.filename.contains(".tar.") ? artifact.filename.find('tar\\..*') : FilenameUtils.getExtension(artifact.filename)
+            String filename = artifact.filename - ("." + ext)
+            def source = 'sdist' == artifact.packageType
```

Comment:
Does `sdist` still belong here?

Reply:
Yep, it just try to figure out if this dep is source dist or not. I can use SOURCE_DIST_PACKAGE_TYPE to replace it.